docs/03-declarative-layout.md

```diff
@@ -40,7 +40,7 @@ configurable rules.
 Code:
 
     s.mywibox : setup {
-        s == 1 and my_first_widget, -- Only display on screen 1
+        s.index == 1 and my_first_widget, -- Only display on screen 1
```

Comment:
@actionless Personally, I find the screen.index notation more powerful, but I can see where other would be useful.

How about I add another widget to that imaginary wibox, one using s == screen.primary, and one using s.index == 2? This would demonstrate both?